client/src/components/ToDoList/ToDoItem/test.js

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
```

Comment:
This file should be named ToDoItem.test.js

Reply:
Also - for consistency, let's stick with using react-test-renderer for snapshot testing. Please refer to PR #70 for how to do that - either wait for that PR to be merged so you can use the package, or install it yourself at the same version so that there is no conflict. Also please make sure that the snapshot file is committed (I don't see a snapshot file in this PR)

Reply:
Oops this file should not be here. It was generated when creating the component. I will delete it later. All tests should be in ToDoList.test.js